tools/src/testcert-gen/main.go

```diff
@@ -0,0 +1,302 @@
+package main
+
+import (
+	"context"
+	crypto_rand "crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math/big"
+	math_rand "math/rand"
+	"net"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"github.com/datawire/dlib/dlog"
+)
+
+func djb2Hash(str string) int64 {
+	hash := uint64(5381)
+	for _, c := range str {
+		hash = (hash * 33) + uint64(c)
+	}
+	return int64(hash)
```

Comment:
Shouldn't this be `uint64`? I guess it doesn't matter if it's just handed off to `math_rand.NewSource` but, hm.

Reply:
Yeah, it's specifically to hand off to `math_rand.NewSource`.  If I were OK with returning a `uint64`, there wouldn't need to be a cast at all.